packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -42,10 +43,17 @@ export function Kubes(props: State) {
     username,
     authType,
     showButton,
-    searchValue,
-    setSearchValue,
+    isEnterprise,
+    isLeafCluster,
+    clusterId,
+    canCreate,
   } = props;
 
+  const [searchValue, setSearchValue] = useState('');
```

Comment:
I would move it back to the hook, no reason to break up a state this way because of 1 value (the next question would be why not move down the rest of the state fields?).